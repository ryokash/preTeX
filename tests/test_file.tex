\documentclass[12pt]{article}
\usepackage{amsmath}
\title{preTeX test file
\begin{document}

\maketitle
% pretex braket_expanding

\section{inline}
This is an epic test file. This should not be translated: x. But some of these: $x. x. y..
b_i.. a. \varphi_i..$. d$\frac a b$. $u_tt$. $a -> b$. $a _ tt ^ a,b$.
$a _ tt ^   a,b$
a$x -> y\$x -> y$b
$a -> b$
$a*b$
$a* b$
$a<bra|mid|ket>b$
$a<bra|ket>b$
$a |\ket> b$
$a <bra| b$
$a |k><bra| b$
$= { x | x>0 }$

\section{single and multiple labels}
\begin{eqnarray}
    x.
    \Sigma(R) = \Sigma_{0} \left(\frac{R}{R_{\rm c}}\right)^{-q}{\rm exp}\left[-\left(\frac{R}{R_{\rm c}}\right)^{2-q}\right],
    a x. a \label{eq_test} x x. x
\end{eqnarray}

\begin{eqnarray}
    x.
    \Sigma(R) = \Sigma_{0} \left(\frac{R}{R_{\rm c}}\right)^{-q}{\rm exp}\left[-\left(\frac{R}{R_{\rm c}}\right)^{2-q}\right],
    a x. a \label{eq_test} x x. x  x x. x \label{eq_test} x x. x
\end{eqnarray}

\section{markdown}
- this is markdown list
- should still work! $a*b$ but not $a^*$ and not here... $x^2 <1|2|3>$.

\section{envs}
\begin {align}
a = b \\
x = y
\end{align}

\begin {align}
a = x = b \\
x = y
\end{align}

\begin {align}
a = b \\
x &= y
\end{align}

\begin {align}
a = b
x = y
\end{align}

\begin {align*}
a &= b.
x &= y
\end{align*}

newline
\end{document}