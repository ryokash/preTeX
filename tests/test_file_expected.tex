\documentclass[12pt]{article}
\usepackage{amsmath}
\title{preTeX test file
\begin{document}

\maketitle
% pretex braket_expanding

\section{inline}
This is an epic test file. This should not be translated: x. But some of these: $\dot{x} \dot{x} \ddot{y}
\ddot{b_i} \dot{a} \ddot{\varphi_i}$. $\displaystyle \frac{a}{b}$. $u_{tt}$. $a \to b$. $a _ {tt} ^ {a,b}$.
$a _ {tt} ^   {a,b}$
a$x \to y\$x \to y$b
$a \to b$
$a\cdot b$
$a\cdot b$
$a\braket{bra|mid|ket}b$
$a\braket{bra|ket}b$
$a \ket{\ket} b$
$a \bra{bra} b$
$a \ket{k}\bra{bra} b$
$= { x | x>0 }$

\section{single and multiple labels}
\begin{eqnarray}
    \dot{x}
    \Sigma(R) = \Sigma_{0} \left(\frac{R}{R_{\rm c}}\right)^{-q}{\rm exp}\left[-\left(\frac{R}{R_{\rm c}}\right)^{2-q}\right],
    a \dot{x} a \label{eq_test} x \dot{x} x
\end{eqnarray}

\begin{eqnarray}
    \dot{x}
    \Sigma(R) = \Sigma_{0} \left(\frac{R}{R_{\rm c}}\right)^{-q}{\rm exp}\left[-\left(\frac{R}{R_{\rm c}}\right)^{2-q}\right],
    a \dot{x} a \label{eq_test} x \dot{x} x  x \dot{x} x \label{eq_test} x \dot{x} x
\end{eqnarray}

\section{markdown}
- this is markdown list
- should still work! $a\cdot b$ but not $a^*$ and not here... $x^2 \braket{1|2|3}$.

\section{envs}
\begin {align}
a &= b \\
x &= y
\end{align}

\begin {align}
a = x = b \\
x = y
\end{align}

\begin {align}
a = b \\
x &= y
\end{align}

\begin {align}
a &= b \\
x &= y
\end{align}

\begin {align*}
a &= \dot{b} \\
x &= y
\end{align*}

newline
\end{document}